src/node_process_methods.cc

```diff
@@ -50,6 +50,8 @@ using v8::Uint32;
 using v8::Uint32Array;
 using v8::Value;
 
+Mutex umask_mutex;
```

Comment:
Can you put this into the `per_process` namespace so it's easier to tell from the code that this global variable is shared across the entire process?

Reply:
@joyeecheung done!